relativepath.conf

```diff
@@ -0,0 +1,29 @@
+# Define an upstream block for the backend server
+upstream backend {
+  server localhost:3001;
+}
+# Define a server block that listens on port 80 and serves requests for the "pi" server_name
+server {
+  listen 80;
+  server_name rp;
+  # Set the document root to the public directory of the relativepath project
+  root /home/jasondoze/relativepath/public;
+  # Define a location block for the root path "/"
+  location / {
```

Comment:
Yeah this is what I was having you try out. This looks right. You need to add the root and then the location.